mods/cnc/rules/structures.yaml

```diff
@@ -6,15 +6,16 @@ FACT:
 	Tooltip:
 		Name: Construction Yard
 	Building:
-		Footprint: xxx xxx
-		Dimensions: 3,2
+		Footprint: xxx xxx ===
+		Dimensions: 3,3
+		LocalCenterOffset: 0,-420,0
```

Comment:
Because of `BodyOrientation.CameraPitch` not being 255 (same 'issue' as with vertical HitSHape.TargetableOffsets). But I just noticed it should actually have been 840 in that case.